app/script/localization/translations/webapp-fr.js

```diff
@@ -393,7 +393,7 @@ z.string.fr.participantDevicesDetailVerify = 'Vérifié';
 z.string.fr.participantDevicesHeader = 'Appareils';
 z.string.fr.participantDevicesHeadline = 'Wire donne à chaque appareil une empreinte unique. Comparez-les avec {{user}} et vérifiez votre conversation.';
 z.string.fr.participantDevicesLearnMore = 'En savoir plus';
-z.string.fr.participantDevicesWhyVerify = 'Pourquoi vérifier une conversation ?';
+z.string.fr.participantDevicesWhyVerify = 'Pourquoi vérifier les conversations ?';
```

Comment:
while we are here.. extra space before `?`

Reply:
This is correct: https://unicode.org/udhr/n/notes_fra.html

Reply:
TIL :)

Reply:
We should maybe replace all the ` ?` during the post process in this file with a non-breaking space in order to avoid having the `?` ending up in a separate line if it can't fit..

(but in a separate PR)